docs/ai_gateway/gateway.md

```diff
@@ -0,0 +1,159 @@
+# Pydantic AI Gateway
+
+**Pydantic AI Gateway** (PAIG) is a unified interface for accessing multiple AI providers with a single key. Features include built-in OpenTelemetry observability, real-time cost monitoring, failover management, and native integration with the Pydantic stack.
```

Comment:
I think having a bullet point with emojis on the feature is nice touch.

(not blocker/required)

Reply:
Ah, I see it's written a bit below.